cmd/kubent/main_test.go

```diff
@@ -15,10 +15,10 @@ func TestInitCollectors(t *testing.T) {
 	testConfig := config.Config{
 		Filenames:  []string{"../../fixtures/deployment-v1beta1.yaml"},
 		Cluster:    false,
-		Debug:      false,
 		Helm2:      false,
 		Helm3:      false,
 		Kubeconfig: "test",
+		LogLevel:   config.ZeroLogLevel(int8(3)),
```

Comment:
just
```suggestion
		LogLevel:   3,
```
should work in this case. Go does implicit type conversion for constants (+no need for double conversion). Maybe a bit nicer:
```suggestion
		LogLevel:   config.ZeroLogLevel(zerolog.ErrorLevel),
```